From d0cc41f94708c98e25285ed341f60d23f4447abe Mon Sep 17 00:00:00 2001
From: Steve Wainstead <wainstead@gmail.com>
Date: Sat, 4 Dec 2010 01:14:20 +0000
Subject: [PATCH 078/223] removing unused variable 'timeout' from
 proto_accept_connection():

    int timeout = server_int_option("name_lookup_timeout", 5);
---
 server/net_bsd_tcp.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/server/net_bsd_tcp.c b/server/net_bsd_tcp.c
index 81d0f40..d53cbb8 100644
--- a/server/net_bsd_tcp.c
+++ b/server/net_bsd_tcp.c
@@ -160,7 +160,6 @@ enum proto_accept_error
 proto_accept_connection(int listener_fd, int *read_fd, int *write_fd,
 			const char **name)
 {
-    int timeout = server_int_option("name_lookup_timeout", 5);
     int fd;
     struct sockaddr_in address;
     socklen_t addr_length = sizeof(address);
-- 
1.7.9.5

